apps/frontend/src/app/components/footer/footer.component.css

```diff
@@ -19,6 +19,12 @@
   list-style: none;
 }
 
+.ekreta img {
+  /* filter: gray; /* IE6-9 */
+  -webkit-filter: grayscale(1); /* Google Chrome, Safari 6+ & Opera 15+ */
```

Comment:
We don't use vendor prefixes, if anything needs prefixing autoprefixer plugin will do that on compile time. Also if you look this up, filter is supported without prefixes since 3 years in evergreen browsers. Lastly, we don't support IE, so you can just leave line 25 and remove the comments

Reply:
Fixed in 2239f82